### DISCUSSION TITLE
XML Export to other Portals

### DISCUSSION THREAD

**Discussion by tarektoubia** [prolinkinfo/RealEstateCRM#71] [General]
Dear Support,

I haven't installed or tested the CRM yet, but the Demo shows how powerful it is.
However, I could not detect any features related to XML data export for third party websites.

So I need to know if it is feasible to export XML data from the CRM and import it to other portals.

Regards

**Comment by prolinkinfo** [comment]
@tarektoubia 

Thanks for the appreciation. 

It has CSV import and export feature. We have not implemented XML for now but it's on our Road map. It could take a few months as we are working on some other important features now.

**Reply by tarektoubia** [reply]
I think CSV can do the job, but is it a Manual CSV Export Feature?

**Reply by prolinkinfo** [reply]
Yes, it's a manual CSV export.
